src/sections/blog-post.liquid

```diff
@@ -0,0 +1,184 @@
+<div class="page-width">
```

Comment:
Maybe we could use the `<section>` tag as a wrapper? No need for the class.

Reply:
`<section>` for sure wrapping most (likely all) of these homepage sections. 

I had a chat with Scott about this PR as there are some general HTML structure questions I had.  I'll put them as a comment on this thread.

Reply:
I use `<section>` now